ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Suchparameter.md

```diff
@@ -59,9 +62,9 @@ Diese Suche gibt alle Condition-Ressourcen zurück zum Client, welche innerhalb
 
 ### Reference
 
-Der Modifier `:identifier` MUSS für alle spezifizierten Suchparameter vom Typ 'Reference' unterstützt werden.
+Der Modifier `:identifier` KANN für alle spezifizierten Suchparameter vom Typ 'Reference' unterstützt werden.
 
-Der [type] Modifier MUSS für alle spezifizierten Suchparameter vom Typ 'Reference' unterstützt werden.
+Der [type] KANN für alle spezifizierten Suchparameter vom Typ 'Reference' unterstützt werden.
```

Comment:
Das MUSS sollte hier bestehen bleiben, da ich an einigen Stellen sonst nicht in die richtige Ressource per Chaining reingehen kann als Client.